include/etcpal/log.h

```diff
@@ -164,16 +229,15 @@ typedef struct EtcPalLogTimeParams
 } EtcPalLogTimeParams;
 
 /*! The set of log strings passed with a call to an etcpal_log_callback function. Any members not
- *  requested in the corresponding EtcPalLogParams struct will be NULL.
- */
+ *  requested in the corresponding EtcPalLogParams struct will be NULL. */
 typedef struct EtcPalLogStrings
 {
   /*! Log string formatted compliant to RFC 5424. */
   const char* syslog;
   /*! Log string formatted for readability per ETC convention. */
   const char* human_readable;
-  /*! The original log string that was passed to etcpal_log() or etcpal_vlog(). Will overlap with one of
-   * syslog_str or human_str. */
+  /*! The original log string that was passed to etcpal_log() or etcpal_vlog(). Will overlap with
+   *  one of syslog_str or human_str. */
   const char* raw;
```

Comment:
We should include the original log priority as an integer here so that you can forward log messages to the syslog daemon by passing the `raw` string and the priority.